controllers/openstackdataplanenodeset_controller.go

```diff
@@ -128,7 +127,11 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
 		// update the Ready condition based on the sub conditions
 		if instance.Status.Conditions.AllSubConditionIsTrue() {
 			instance.Status.Conditions.MarkTrue(
-				condition.ReadyCondition, dataplanev1.DataPlaneRoleReadyMessage)
+				condition.ReadyCondition, condition.ReadyMessage)
+		}
+		// update the overall status condition if service is ready
+		if instance.IsReady() {
+			instance.Status.Conditions.MarkTrue(condition.ReadyCondition, dataplanev1.DataPlaneNodeSetReadyMessage)
```

Comment:
I'm not sure this looks right. Why the further changes from e40961c9a99343422671ada69c0ddcb4e2a6101c?

Reply:
eh yeah, I fixed it again.